app/controllers/three_d_secure_controller.js

```diff
@@ -52,7 +52,12 @@ const handleThreeDsResponse = (req, res, charge) => response => {
   switch (response.statusCode) {
     case 200:
     case 400:
-      redirect(res).toConfirm(charge.id)
+      logger.info('3DS required again', getLoggingFields(req))
```

Comment:
It looks like this log should only be called inside of the first conditional?

Reply:
Whoops!